src/project/internal/notationproject.cpp

```diff
@@ -290,6 +290,15 @@ mu::Ret NotationProject::doImport(const io::path_t& path, const io::path_t& styl
     mu::engraving::ScoreLoad sl;
     m_engravingProject->setFileInfoProvider(std::make_shared<ProjectFileInfoProvider>(this));
     mu::engraving::MasterScore* score = m_engravingProject->masterScore();
+
+    // Load style if present
+    if (!stylePath.empty()) {
+        score->loadStyle(stylePath.toQString());
+    }
+
+    // Init ChordList
```

Comment:
Shouldn't this be after setupMasterScore? If the order is important here, you should specify it in the comment